code/modules/religion/sparring/ceremonial_gear.dm

```diff
@@ -25,6 +25,10 @@
 	material_flags = MATERIAL_EFFECTS | MATERIAL_ADD_PREFIX | MATERIAL_GREYSCALE //doesn't affect stats of the weapon as to avoid gamering your opponent with a dope weapon
 	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, FIRE = 100, ACID = 50)
 	resistance_flags = FIRE_PROOF
+	//SKYRAT ADDITION START
+	wound_bonus = CANT_WOUND
```

Comment:
can you explain why this needs to be done

Reply:
Xyel wanted this done because they wanted the chaplain's sparring sect to not be nearly as lethal